src/plots/cartesian/axes.js

```diff
@@ -1166,38 +1223,44 @@ axes.calcTicks = function calcTicks(ax, opts) {
 
     tickVals = tickVals.concat(minorTickVals);
 
-    var t, p;
+    function setTickLabel(ax, tickVal) {
+        var t = axes.tickText(
```

Comment:
Please rename `t` variable inside this function as there is another `t` declared in the upper scope.